requirements.txt

```diff
@@ -5,7 +5,7 @@ Flask-Script
 Flask-Migrate
 nose
 nose-cov
-psycopg2
+psycopg2-binary==2.8.6
```

Comment:
Glad we got this added in. The errors I got from using the other method are not pretty!!
[](https://media3.giphy.com/media/l44QrV0bFfDkhjD8Y/giphy.gif)

Reply:
Thanks for asking Ian about this!